docs/source/quickstarter.rst

```diff
@@ -71,53 +71,37 @@ This will run all functions required to create the output specified in the `run.
     $ python run.py
 
 
-Run using the `DAGWorks Platform <app.dagworks.io>`_
-----------------------------------------------------
+Track execution with the `Hamilton UI <https://github.com/dagworks-inc/hamilton/tree/main/ui>`_
+___________________________________________________________________________________________________________
+If you would like to track the execution of the `naturf` workflow, there is an interactive UI
+available that allows you to track the progress of the workflow, view logs, and capture summary statistics of outputs.
 
-Set the DAGWorks API Key as an environment variable:
+1. Pre-requisites:
 
-.. code:: bash
-
-    $ export DAGWORKS_API_KEY="<your API Key>"
+* Have the self-hosted Hamilton UI running and you have created a user and project. If not, follow the instructions in the `Hamilton UI README <https://github.com/dagworks-inc/hamilton/tree/main/ui>`_.
+* Or, have a free account on `DAGWorks Inc. <https://www.dagworks.io/hamilton>`_, and have created a project and an API Key.
+* Have the right SDK installed. If not, install it using the following command:
 
+    .. code:: bash
 
-Start with the `run.py` file from above (using either the example data or your own data) and add the following. Import os and DAGWorks adapters, which contains the tracker:
-
-.. code:: python3
+        $ pip install sf-hamilton[sdk]  # if self-hosting the Hamilton UI
+        $ pip install dagworks-sdk  # if using the hosted Hamilton UI via DAGWorks Inc.
 
-    import os
-    from dagworks import adapters
 
+2. Set the requisite environment variables:
 
-Initialize the DAGWorks tracker:
```

Comment:
I removed this part because this is already included in the driver.py file -- and the user needn't do much.